### ISSUE TITLE
Support source path with whitespace

### ISSUE THREAD

**Issue by SCullman** [Zaid-Ajaj/Femto#67]
My project path includes a space, causing femto to fail:
```
PS C:\Users\Firstname Lastname\spikes\project\src\client> dotnet femto install Feliz
[06:46:10 INF] Analyzing project C:\Users\Firstname Lastname\\spikes\project\src\client\Client.fsproj
[06:46:10 INF] Running dotnet restore against the project
[06:46:12 INF] Detected paket.references file -> use paket
[06:46:12 INF] Installing Feliz within dependency group Client
[06:46:12 INF] Using locally installed paket
[06:46:12 ERR] Error while running the following command:
[06:46:12 ERR] C:\Users\Firstname Lastname\spikes\project\src\client> dotnet paket add Feliz --project C:\Users\Firstname Lastname\spikes\project\src\client\Client.fsproj --group Client
[06:46:12 ERR] Process Output:
[06:46:12 ERR] Process Error: ... "dotnet-paket add Feliz --project C:\Users\Firstname" not found...
```

**Comment by Zaid-Ajaj** [comment]
Hi @SCullman, it looks like the path need to be escaped if there is a space in it, maybe it should be escaped anyways 🤔 I will look into it

**Comment by Zaid-Ajaj** [comment]
Should be fixed as of v0.8.6 😄 can you please update and verify that the issue is resolved? thanks!

**Comment by SCullman** [comment]
Yes, it works now. Thanks
